azurelinuxagent/common/event.py

```diff
@@ -71,11 +71,11 @@ def save_event(self, data):
         except IOError as e:
             raise EventError("Failed to write events to file:{0}", e)
 
-    def add_event(self, name, op="", is_success=True, duration=0, version="1.0",
+    def add_event(self, name, op="", is_success=True, duration=0, version=AGENT_VERSION,
```

Comment:
@ahmetalpbalkan Agreed. It is too large a change for now. The prior value was "1.0"...an obviously incorrect value.